examples/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.12)
+
+set(PICO_SDK_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../lib/pico-sdk)
+include(${PICO_SDK_PATH}/external/pico_sdk_import.cmake)
+
+project(pico_ice_examples C CXX ASM)
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
+pico_sdk_init()
+
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/pico-ice-sdk)
+
+add_subdirectory(pico_blinky)
+add_subdirectory(pico_blinky_cplusplus)
+add_subdirectory(pico_cram)
+# colides with pico sdk target
+#add_subdirectory(pico_flash)
```

Comment:
That is right, this needs fixing. Maybe an `example_pico_...` and `example_ice_...` prefix is the way.
Thank you for raising this.